image/base/runit-1.d-00-fix-var-logs

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 mkdir -p /var/log/nginx
 chown -R www-data:www-data /var/log/nginx
-chown www-data:www-data /var/log/nginx
```

Comment:
Yes... It's completely redundant given line 3.  `/var/log/nginx` and it's descendents are chown'd, so all line 4 does is reïterate that `/var/log/nginx` is supposed to be chown'd.